projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/palette.module.ts

```diff
@@ -251,7 +245,9 @@ const PALETTE_COMPONENTS = [
     MatDialogModule,
     MediaViewerModule,
     LoadingModule,
-    RpxTranslationModule.forChild()
+    RpxTranslationModule.forChild(),
+
+    QueryManagementModule
```

Comment:
@DavidJayakumar - why is `QueryManagementModule` imported here? We haven't done this for anything else, e.g. Case Flags, Case Linking, Hearings, etc.

Reply:
Instead of importing and exporting the individual components, it is better to use module. But I will check with @ronaldmansveld

Reply:
Happy to do it via module